plugins/nf-hello/build.gradle

```diff
@@ -78,10 +78,43 @@ dependencies {
     modules {
         module("commons-logging:commons-logging") { replacedBy("org.slf4j:jcl-over-slf4j") }
     }
+
+    // Add this line to explicitly declare the test framework launcher
+    testRuntimeOnly 'org.junit.platform:junit-platform-launcher'
 }
 
 // use JUnit 5 platform
 test {
     useJUnitPlatform()
 }
 
+// Task to install the plugin to the Nextflow plugins directory
```

Comment:
Would it make sense to put this task in the parent `build.gradle`, inside the `submodules` config with the other related tasks?

Reply:
It would make sense if we can have the task iterate over all the possible plugins in the plugins directory. There is only a single plugin at the moment, but it's very possible to have a single repository that hosts multiple plugins.
I'm no gradle expert, but I'll see what I can do.

Reply:
Done. Thanks for the suggestion, Tom!